main/templates/user/user_list.html

```diff
@@ -97,13 +97,13 @@
       </thead>
       <tbody>
         # for user_db in user_dbs
-          <tr id="{{user_db.key.urlsafe()}}" class="{{'text-muted' if not user_db.active}}">
+          <tr id="{{user_db.key.urlsafe()}}" class="row-link {{'text-muted' if not user_db.active}}" data-href="{{url_for('user_update', user_id=user_db.key.id())}}">
             <td class="name">
-              <label>
+              <label class="not-link">
                 <input type="checkbox" name="user_db" value="{{user_db.key.urlsafe()}}">
                 <img src="{{user_db.avatar_url_size(80)}}" alt="Avatar of {{user_db.name}}">
               </label>
-              <a href="{{url_for('user_update', user_id=user_db.key.id())}}">{{user_db.name}}</a>
+              <span>{{user_db.name}}</span>
```

Comment:
Because if you don't it looks like this
![screen shot 2015-03-21 at 22 01 32](https://cloud.githubusercontent.com/assets/125676/6766816/13070e10-d016-11e4-96bf-70933451d69e.png)

Instead of
![screen shot 2015-03-21 at 22 01 20](https://cloud.githubusercontent.com/assets/125676/6766821/1d9337be-d016-11e4-893c-2829f5d80920.png)